scripts/cromwell_status_parser.py

```diff
@@ -101,6 +175,8 @@ def copy_outputs_to_bucket(
     source_bucket = storage_client.bucket(source_bucket_name)
     destination_bucket = storage_client.bucket(destination_bucket_name)
 
+    print(f'{dataset} :: {sg_id} :: Copying outputs summary:')
+    print(f'  Destination: gs://{destination_bucket_name}/sv_evidence/\n')
     for _, output in outputs.items():
         for _, value in output.items():
```

Comment:
instead of `for _, x in y.items()` you can just drop down to `for x in y.values()`